### ISSUE TITLE
Restricting access to certain fields for public editors

### ISSUE THREAD

**Issue by campgurus** [BOWiki/BOW#369]
We may want to restrict access to fields we don't want changed. e.g. offensive things could happen if the wrong person decides to change out the article image.

**Comment by trystant** [comment]
I think that we need to think over how we want to do this.

If I were you, I'd check out discourse.org about trust levels and how a
person can gain trust and then gain access to do certain things.

When I say "check out", I mean, go to discourse.org, take a look at the
software, sign up for an account at meta.discourse.org, log in and check
out the way that the software works.



On Mon, Feb 29, 2016 at 4:10 AM, campgurus <notifications@github.com> wrote:

> We may want to restrict access to fields we don't want changed. e.g.
> offensive things could happen if the wrong person decides to change out the
> article image.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BOWiki/BOW/issues/369>.
>



-- 

Mark Nyonmark@grandkru.com

blog.grandkru.com

github.com/trystant

**Comment by campgurus** [comment]
I ma happy to review, but on initial glance, this appears to be focused on policing content on comment forums which will eventually be necessary. However, right now I am thinking more in terms of not allowing rogue users to post d--k pics instead of images of victims.

**Comment by trystant** [comment]
No, the central concept of trust level applies to whatever content. Which
actions are available or not can vary, but as more people get involved, we
can grant the ability to change certain settings or not based upon the
trust level.

On Mon, Feb 29, 2016 at 10:36 AM, campgurus <notifications@github.com>
wrote:

> I ma happy to review, but on initial glance, this appears to be focused on
> policing content on comment forums which will eventually be necessary.
> However, right now I am thinking more in terms of not allowing rogue users
> to post d--k pics instead of images of victims.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BOWiki/BOW/issues/369#issuecomment-190259354>.
>



-- 

Mark Nyonmark@grandkru.com

blog.grandkru.com

github.com/trystant

**Comment by trystant** [comment]
Revisiting this. How important is this in the scheme of things, and do we have a strategy about how to execute? I still think the discourse.org trust model is very appropriate here, but what is the priority on this.
